README.md

```diff
@@ -117,6 +117,16 @@ jf 'rt u target/ my-repo/'
 
 > **_IMPORTANT:_** Notice the single quotes wrapping the command right after the **jf** step definition.
 
+If the JFrog CLI command contains some arguments with spaces, you can rather provide the arguments as a list:
+```groovy
+jf(['mvn', 'clean', 'install', '-Ddeploy.testProperty=Property with space'])
+```
+In this case, the quotes that would be required in a shell are not necessary (and would be part of the value itself, if used). 
+The above step is equivalent to a shell invocation as follows:
+![bash-spaces.png](images/readme/bash-spaces.png)
+
+Passing the arguments as a list can also be useful to avoid space and escaping problems if some of those arguments rely on script variables.
```

Comment:
Passing the arguments as a list can also be useful to avoid space and escaping problems if some of those arguments rely on script variables.
-->
The list syntax also helps avoiding space and escaping problems, when some of those arguments use script variables.